tests/web3js/eth_transaction_validations_test.js

```diff
@@ -0,0 +1,109 @@
+const { assert } = require('chai')
+const conf = require('./config')
+const helpers = require('./helpers')
+
+it('validates transactions', async () => {
```

Comment:
I prefer having a unit test using the `validate()` function directly, this test adds complexity of setting up e2e testing for the sake of just passing in different formats.

Reply:
You're right, I missed that alternative :sweat_smile: 
Removed the E2E test in https://github.com/onflow/flow-evm-gateway/pull/314/commits/4af431599eb85248f5d94c0ed34e3844782392e8
Added the unit tests in https://github.com/onflow/flow-evm-gateway/pull/314/commits/a851720798cbdaec7775d78c45941df070d25134